src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Colors color = colorSupplier.getRandomColor();
+    private Random random = new Random();
+    private int number = random.nextInt(100);
```

Comment:
What is it? All you need here is a number field and a color. But they don't need to be defined either, for that use a contrunctor or setters